.gitignore

```diff
@@ -1,15 +1,7 @@
-.idea
-*apache/
+*.idea
+*.iml
+*.idea/
 *.gradle/
-*/build/
-results/
-*.csv
-*.ipynb
-RefactoringsExperiments/*
-!RefactoringsExperiments/data/
-!RefactoringsExperiments/gradlew.bat
-!RefactoringsExperiments/log4j2.xml
-!RefactoringsExperiments/gradlew
-!RefactoringsExperiments/build.gradle
-!RefactoringsExperiments/src/
-!RefactoringsExperiments/gradle/
\ No newline at end of file
+*build/
+clones/
+out/
```

Comment:
add build folders from the modules
`extract-method-plugin/build/` and others

Reply:
should we specifically ignore each folder one-by-one if single regular expression can catch them?

Reply:
oh I see, `build/` works for any level and there is no need to clarify each folder separately?

Reply:
great, thanks!